src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -210,7 +210,7 @@ foam.CLASS({
             data:   this.data,
             config: this.config,
             of:     this.config.of
-          }, parent: this.__subContext__.createSubContext({ memento: this.memento })
+          }, parent: this.__subContext__.createSubContext({ memento_: this.memento_ })
```

Comment:
This shouldn't be required at all, are we sure we need this?

Reply:
not needed on the dashboard widget and tested on DAOUpdateView working fine without passing memento from here.